release-team/role-handbooks/branch-manager/README.md

```diff
@@ -1,235 +1,307 @@
+   - [ ] [kubernetes-release-team](https://groups.google.com/forum/#!forum/kubernetes-release-team)
+   - [ ] [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev)
+
 
 [k-announce-list]: https://groups.google.com/forum/#!forum/kubernetes-announce
 [k-announce-request]: https://groups.google.com/forum/#!contactowner/kubernetes-announce
-[release tooling]: https://github.com/kubernetes/release
+[release tools]: https://github.com/kubernetes/release
+
+## Safety Check
 
-# Safety Check
-Run the following command to affirm your release repo checkout is in a good state, your development client is fully configured, and your user identity is authorized for the builds:
-* `./gcbmgr`
+The following command can help verify that your clone of the [release tools] are up-to-date, that your setup is fully configured, and your `gcloud` identity is authorized for the release builds:
+
+`./gcbmgr`
    * should run successfully and even show some green colored "OK" words
-* `./gcbmgr list`
-   * should see for example: 1.9, 1.10, 1.11 builds
 
-## First Alpha Mock Build
+# The Release Process
+
+**General overview**:
+
+Public build artifacts are published and an email notification goes out to the community. You will become very familiar with the following commands over the course of the 3 month release cycle:
+
+```bash
+./gcbmgr stage
+./gcbmgr release ${{BRANCH_NAME}} --buildversion=${{VERSIONID}} --nomock
+# Only for the official release: Inform the Google team to complete the corresponding Deb and RPM builds
+./release-notify vX.Y.0-{alpha,beta,rc}.Z
+./release-notify vX.Y.0-{alpha,beta,rc}.Z --nomock --mailto=${{YOUREMAILADDRESS}}[a][b]
+```
+
+There are more examples of the release process under the [References](#references) section.
+
+It is also highly recommended to glance over the [Release Commands Cheat Sheet](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/patch-release-manager#release-commands-cheat-sheet)
+
+**General bookkeeping**:
+
+Prior to starting the release process, [open a Release Cut issue](https://github.com/kubernetes/sig-release/issues/new?template=cut-release.md&title=Cut+1.x.y-%7Balpha%2Cbeta%2Crc%7D.z+release) on [k/sig-release](https://github.com/kubernetes/sig-release)
+
+In the Release Cut issue template, there comments describing the are predefined items to complete during the release process.
+
+On the item `Collect metrics, links...`:
+
+You can make a copy of this spreadsheet to collect metrics of build times and include links to the GCP build console:
+
+[Branch Management Log](https://docs.google.com/spreadsheets/d/1gbr2byg7AmGvVv6onZtgowa729KZcyYTyH82qn1czK8/edit?usp=sharing)
+
+A section of the log ⬆️ is usually copied over as a comment on the release issue. See [this thread](https://github.com/kubernetes/sig-release/issues/603#issuecomment-487735684) for cutting [1.15.0-alpha.2](https://github.com/kubernetes/sig-release/issues/603).
```

Comment:
Can this say "above" instead of an arrow icon?  I suspect that will be more accessible.